metacluster/include/metacluster/MetaclusterUtil.actor.h

```diff
@@ -38,6 +40,8 @@
 // This provide metacluster utility functions that may be used both internally and externally to the metacluster project
 namespace metacluster::util {
 
+FDB_BOOLEAN_PARAM(SkipMetaclusterCreation);
```

Comment:
Is this used only in simulation test? If so, I wonder whether we can put the test-only types `SimulatedMetacluster`, `SkipMetaclusterCreation`, variables, functions in a dedicated nested namespace.

Reply:
The util namespace is already meant for special purpose things like this. At some point we may have enough that we want to further divide it, and I think for now it's probably ok to have this in the top-level util namespace.